vmdb/app/helpers/container_node_helper/textual_summary.rb

```diff
@@ -97,6 +97,22 @@ def textual_container_groups
     h
   end
 
+  def textual_lives_on
+    lives_on_ems = @record.lives_on.try(:ext_management_system)
+    return nil if lives_on_ems.nil?
+    # TODO: handle the case where the node is a bare-metal
+    {
+      :label => "Underlying Host",
```

Comment:
Ah actually I have to build it intelligently anyway as it could be "Instance" (cloud) or "Virtual Machine" (infra).